Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,89 @@
 
 import Foundation
 
-print("Hello, World!")
+
+var chances = 6
+let allTheWords = ["apple"]
```

Comment:
Initially I thought something was off with the way your game was resetting but the error is here, your array of allTheWords only has one word in it, so the user can only guess one word.